docs/how-to/ocm-control-plane-walkthrough.md

```diff
@@ -37,6 +35,12 @@ We will start with a single cluster and move to multiple clusters to illustrate
     AWS_REGION=eu-west-1
     ```
 
+* Add some env-vars to your `zsh.rc` or `.bash_profile`
```

Comment:
It might be that I've run through the walkthrough so many times, but I tend to use a different app host (same subdomain) each time.
So I would probably just export it in the session as opposed to setting it statically in rc or profile. e.g. `export MGC_SUB_DOMAIN=ocmtest01.dm.hcpapps.net`

However, I'd have to export it in each terminal if that was the case. 
I guess that's why you're suggesting this approach?

Reply:
yeah, agreed - I'll make that clearer, I do something similar to you (override often, but have a default)

Reply:
Updated, tried to make this a little clearer